Examples/README.md

```diff
@@ -3,8 +3,6 @@ This project contains examples of how to use the toolkit components.
 ## Instructions
 
 
- 1. Ensure you have downloaded and installed __ArcGIS Runtime SDK for iOS__ as described [here](https://developers.arcgis.com/ios/get-started/)
- 2. Clone or download this repo. 
- 3. Open the Examples project in Xcode by double clicking on the `ArcGISToolkitExamples.xcodeproj` file
- 4. Build and run the project. 
-
+ 1. Clone or download this repo
+ 2. Open the Examples project in Xcode by double clicking on the `ArcGISToolkitExamples.xcodeproj` file
+ 3. Build and run the project. 
```

Comment:
Bullet points with only one sentence don't need a period... ;-)
```suggestion
 3. Build and run the project
```

Reply:
ah you exposed me, I fell into my own trap 😅